app/views/merge_requests/confirm_telephone_number.html.erb

```diff
@@ -8,6 +8,30 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds-from-desktop">
-    <p class="govuk-body">Confirm the telephone number on file, or enter a new one.</p>
+    <% if @merge_request.absorbing_organisation.phone.present? %>
+      <p class="govuk-body">Confirm the telephone number on file, or enter a new one.</p>
+    <% end %>
+
+    <div class="govuk-inset-text">
+      <%= @merge_request.absorbing_organisation.phone %>
+    </div>
```

Comment:
Do we want to display this if the phone is empty? It has an empty inset text in that case